scenario-api/src/main/scala/pl/touk/nussknacker/engine/graph/node.scala

```diff
@@ -325,6 +325,7 @@ object node {
         initialValue: Option[FixedExpressionValue],
         hintText: Option[String],
         valueEditor: Option[ParameterValueInput],
+        // TODO: Research if this parameter may be removed after merged: https://github.com/TouK/nussknacker/pull/7574
```

Comment:
We already know that it can't be removed because this feature is needed. Didn't we?